src/scenes/components/Speakers/components/Speaker/Speaker.js

```diff
@@ -0,0 +1,86 @@
+import React from 'react'
+import styled from 'styled-components'
+import PropTypes from 'prop-types'
+import { H3, Body2 } from '../../../../../components/typography'
+
+const Root = styled.div`
+  padding: 5px;
+`
+
+const Photo = styled.img`
+  border-radius: 50%;
+  border: 2px solid ${props => props.theme.color()};
+  width: 150px;
+  
+  ${({ theme }) => theme.mixins.mediaQuery.tablet`
+    width: 100px;
+  `};
+
+  ${({ theme }) => theme.mixins.mediaQuery.phone`
+    width: 70px;
+  `};
+`
+
+const Name = styled.div`
+    padding: 10px 0;
+`
+
+const Job = styled.div`
+  text-align: center;
+  color: ${props => props.theme.color('neutral', 'base')};
+`
+
+const Link = styled.a`
+  color: #535353;
```

Comment:
Can we use `palettes.neutral.base` here?

Reply:
We can and we should! My bad